### PR TITLE
refactor: drop some styles in slider

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/demo/src/modules/components/slider/examples/2/index.less
Comment (nsbarsukov) [inline projects/demo/src/modules/components/slider/examples/2/index.less:1]: Just single color:

```less
.first {
    color: var(--tui-support-01);
}

.second {
    color: var(--tui-support-03);
}

.third {
    color: var(--tui-support-12);
}
```

File: projects/kit/components/slider/slider.style.less [hunk 1]
```diff
@@ -157,8 +142,7 @@
     background-clip: content-box; /* To clip filling of paddings */
     outline: none;
     border-radius: var(--tui-radius-m);
-    // TODO: Uncomment it in v4.0
-    // margin: 0; /* Drop browser default margins */
+    margin: 0;
```
Comment (nsbarsukov) [inline projects/kit/components/slider/slider.style.less:145]:
```suggestion
    margin: 0; /* Drop browser default margins */
```

File: projects/demo/src/modules/components/slider/examples/2/index.less
Comment (splincode) [reply projects/demo/src/modules/components/slider/examples/2/index.less:1]: fixed

File: projects/kit/components/slider/slider.style.less (hunk 1 above)
Comment (splincode) [reply projects/kit/components/slider/slider.style.less]: fixed

File: projects/kit/components/slider/slider.style.less [hunk 2]
```diff
@@ -157,8 +142,7 @@
     background-clip: content-box; /* To clip filling of paddings */
     outline: none;
     border-radius: var(--tui-radius-m);
-    // TODO: Uncomment it in v4.0
-    // margin: 0; /* Drop browser default margins */
+    margin: 0; /* Drop browser default margins */
```
Comment (nsbarsukov) [inline projects/kit/components/slider/slider.style.less:145]: Delete this lines too in this PR (to see that all screenshots are still fine).

https://github.com/taiga-family/taiga-ui/blob/34e421528407c395e6cbad4fa2bb24902075f898/projects/kit/components/range/range.style.less#L131-L132

I've forgotten about it during the first review, sorry!

Comment (splincode) [reply projects/kit/components/slider/slider.style.less]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**